src/lib_ccx/ccx_encoders_common.c

```diff
@@ -1045,203 +1041,204 @@ int encode_sub(struct encoder_ctx *context, struct cc_subtitle *sub)
 		// Write to a buffer that is later s+plit to generate split
 		// in sentences
 		if (sub->type == CC_BITMAP)
-			wrote_something = write_cc_bitmap_to_sentence_buffer(sub, context);
+			sub = reformat_cc_bitmap_through_sentence_buffer(sub, context);
+
+		if (NULL==sub)
+			return wrote_something;
 	}
-	else
+	// Write subtitles as they come
```

Comment:
There is a lot of changes lines. BUT, the most changes - deleted <TAB> symbol. Just ignore whitespaces here:

https://github.com/CCExtractor/ccextractor/pull/491/files?w=1